sospice/catalog/file_metadata.py

```diff
@@ -345,6 +345,21 @@ def get_observer(self):
         )
         return observer
 
+    def mid_time(self):
+        """
+        Find "middle time" for observation in FileMetaData
+
+        Return
+        ----------
+        pd.Timestamp
+            The middle time of the observation.
+        """
+        if self.metadata["DATE-BEG"] and self.metadata["TELAPSE"]:
```

Comment:
I don't think that these checks are needed (we can assume these keys to exist, as their presence is checked by `FileMetadata.validate()` (by comparing to `required_columns`)